pages/create-issue.tsx

```diff
@@ -45,14 +47,18 @@ export default function PageCreateIssue() {
 
   }
 
+  const title_and_description_validation = () => {
+    return (issueTitle.length && issueDescription.length) ? false : true
```

Comment:
This is redundant as `issueTitle.length && issueDescription.length` already returns a `boolean`

```
return !(issueTitle.length && issueDescription.length)
```

Reply:
done